src/Stratis.Features.SQLiteWalletRepository/TransactionsToLists.cs

```diff
@@ -59,17 +60,20 @@ public override void RecordReceipt(HashHeightPair block, Script pubKeyScript, Tx
                 OutputIndex = outputIndex,
                 Value = txOut.Value.Satoshi,
                 IsChange = isChange ? 1 : 0
-            });
+            };
+
+            this.conn.Repository.logger.LogTrace("Recording receipt: {0}", output.ToString());
```

Comment:
All logs like these should `debug`.

```suggestion
            this.conn.Repository.logger.LogDebug("Recording receipt: {0}", output.ToString());
```